autoscheduler/frontend/src/components/SchedulingPage/SchedulePreview/ScheduleListItem/ScheduleListItem.tsx

```diff
@@ -82,9 +57,6 @@ const ScheduleListItem: React.FC<ScheduleListItemProps> = ({ index }) => {
                 {`Schedule ${index + 1}`}
               </span>
             </div>
-            <Typography className={styles.gpa} variant="subtitle2">
```

Comment:
Since we're no longer using the `.gpa` class, can you also remove it from the CSS file?